test/__tests__/data/adobe_analytics.json

```diff
@@ -1010,7 +1010,6 @@
         },
         "messageId": "1578564113557-af022c68-429e-4af4-b99b-2b9174056383",
         "properties": {
```

Comment:
what's this test case change for ?
There's no test case for the newly added change - where null or undefined ones should be empty in the output xml payload.

Reply:
Yes, I handled it in this change
<transaction_id> is mapped to either `properties.transactionId` or `properties.order_id` so it is not showing up in the final XML now as both these properties are not there, and it is not coming as undefined as it is being handled from the `removeUndefinedAndNullValues` function.

Reply:
how come purchaseId was mapped to `1234` before ? what's the mapping rule of purchaseId

Reply:
`purchaseId`  is mapped to `properties.purchaseId` OR `properties.order_id` , since we had order_id in properties, hence the value 1234

Reply:
why all these multiple fields mapping support ? Are we not following ecom spec here ? cc: @Gauravudia

Reply:
No @krishna2020 we are not, we have some destination specific customisations too like "override fields" in adobe

Reply:
Resolving this conversation for now for merging this PR. 🙇